doc/tools/coqrst/coqdomain.py

```diff
@@ -29,6 +28,8 @@
 from docutils.parsers.rst.roles import code_role #, set_classes
 from docutils.parsers.rst.directives.admonitions import BaseAdmonition
 
+from itertools import chain
+
```

Comment:
No need to move this import

Reply:
Just alphabetizing the package names.  The OCD kicking in.

Reply:
The convention is to put stdlib packages before external ones